.gitignore

```diff
@@ -4,3 +4,4 @@
 .ijwb
 *.iml
 /.bazelrc.user
+MODULE.bazel.lock
```

Comment:
What's this file? Is it being created by your editor?

Reply:
bzlmod has a lockfile which afaiui the primary feature is it allows you to build offline. but it is pretty verbose, and i believe there are some cross-platform holes in it today, so it adds some maintenance burden to check in unless you really need it. ignoring it allows you the benefits locally w/o having to deal with updating it